src/prefect/cli/project.py

```diff
@@ -87,6 +103,46 @@ async def init(
         key, value = field.split("=")
         inputs[key] = value
 
+    if not recipe:
```

Comment:
We'll also want to check if the current shell is interactive with `is_interactive()` so that this is usable I'm not interactive shells.

Reply:
Makes sense!